influxdb_connector/influxdb_unittests.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+import unittest
+import os
+import argparse
+from influxdb_client import InfluxDBClient, Point, WritePrecision
+from influxdb_client.client.write_api import SYNCHRONOUS
+from influxdb_connector import influxdb_query, custom_bucket_formatting
+
+
+class TestInfluxdb(unittest.TestCase):
+    def setUp(self) -> None:
+        self.args = []
+        # read text file containing required arguments on each line
+        with open('arguments.txt') as f:
```

Comment:
I do not see arguments.txt ? 

I am trying to run the tester but missing files

Reply:
setUP is updated to read the arguments from test_config.yaml